app/models/BannerTests.scala

```diff
@@ -75,6 +75,7 @@ case class BannerTest(
     minArticlesBeforeShowingBanner: Int,
     userCohort: UserCohort,
     locations: List[Region] = Nil,
+    contextTargeting: Option[PageContextTargeting] = Some(PageContextTargeting(Nil,Nil,Nil,Nil)),
```

Comment:
Oh! I'm curious as to why the default here isn't `None` given that it's an `Option`?

Reply:
Good question. I've just changed this to not be an Option.
We can use the default value to handle any records in dynamodb that do not have the field.
And the client can expect this field to always exist